python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -537,7 +572,8 @@ class RandomForestRegressor(Base):
         return predicted_result
 
     def predict(self, X, predict_model="GPU",
-                algo='BATCH_TREE_REORG', convert_dtype=True):
+                algo='BATCH_TREE_REORG', convert_dtype=True,
+                concat_mod_bytes=[]):
```

Comment:
Same as above, we don’t want to add an obscure parameter that is also not clearly named to the main api